### ISSUE TITLE
XMP data from the DJI jpg files

### ISSUE THREAD

**Issue by shahvi** [ianare/exif-py#111]
Hi,
I'd like to read the RelativeAltitude which is in the XMP tags for jpg files from DJI drones.  Is there a way for me to do that using this tool?

Thanks,
Vik

**Comment by pierotofy** [comment]
Yes, but you need to write some custom code to parse XML. See an example at https://github.com/OpenDroneMap/ODM/blob/master/opendm/photo.py#L154
